src/Framework/AspNetCoreAnalyzers/src/Analyzers/Infrastructure/WellKnownTypeData.cs

```diff
@@ -0,0 +1,203 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.App.Analyzers.Infrastructure;
+
+internal static class WellKnownTypeData
+{
+    public enum WellKnownType
```

Comment:
Does this need to be nested?

Reply:
Not necessarily. It was largely a stylistic choice to put the associated data in the same class. To minimize the code delta, I ended up using an alias in the existing analyzers/codefixers but it felt nicer to capture these types in class instead of having them on the top-level namespace.

Reply:
Wouldn't leaving this unnested minimize the code delta by removing the `using WellKnownType = WellKnownTypeData.WellKnownType;`?